### PR TITLE
Improved the error message and comments in hooks/pre-commit, and the …

### PR DESCRIPTION
**Pull request by ryancwalsh**
…newlines work properly at least on Mac

### CODE REVIEW

**Comment by ryancwalsh** [comment]
I just confirmed that newlines work properly on Ubuntu too.

**Comment by ryancwalsh** [comment]
I don't see a problem with linebreaks anymore. Do you? I've tested Mac and Ubuntu. If you notice a problem, please try to capture any details about your platform, what files you touched and how, etc so we can narrow it down.

I'm curious what you think of https://github.com/NEARFoundation/near-contract-tools/pull/21/files

**Comment by encody** [comment]
The newline issue was fixed in https://github.com/NEARFoundation/near-contract-tools/commit/a963a4fc7f12b9983903e39523470b99d13d9249

**Comment by encody** [comment]
> I'm curious what you think of https://github.com/NEARFoundation/near-contract-tools/pull/21/files

Does ESLint fix Rust issues?

File: hooks/pre-commit [hunk 1]
```diff
@@ -1,10 +1,24 @@
...
+#   "editor.formatOnType": true,
+#   "editor.codeActionsOnSave": {
+#     "source.fixAll.eslint": true
+#   },
+#   "[rust]": {
+#     "editor.formatOnSave": true,
+#     "editor.defaultFormatter": "rust-lang.rust-analyzer"
+#   },
+#   //...
+# }
```
Comment (encody) [inline hooks/pre-commit:19]: Could go in `/.vscode/` or README; this is not a place most people would look for development environment configuration instructions.

File: hooks/pre-commit [hunk 2]
```diff
@@ -1,10 +1,24 @@
...
+# In the project directory, run `git config core.hooksPath hooks/` to install the hooks of this directory (without affecting how git hooks work for other projects).
+
+# To avoid having this pre-commit hook block your commits, configure your VSCode like this:
+
+# {
+#   "editor.formatOnPaste": true,
+#   "editor.formatOnSave": true,
+#   "editor.formatOnType": true,
+#   "editor.codeActionsOnSave": {
+#     "source.fixAll.eslint": true
```
Comment (encody) [inline hooks/pre-commit:12]: Does ESLint fix Rust issues?

**Review by encody — CHANGES_REQUESTED** [review]

**Comment by ryancwalsh** [comment]
> The newline issue was fixed in [a963a4f](https://github.com/NEARFoundation/near-contract-tools/commit/a963a4fc7f12b9983903e39523470b99d13d9249)

@encody  Nice!

I'll try creating a .vscode file now to clean up the rest.

**Comment by ryancwalsh** [comment]
@encody I liked your idea and think https://github.com/NEARFoundation/near-contract-tools/pull/21/files is what you were talking about.

**Review by encody — APPROVED** [review]

**Pull request merged**